README.md

```diff
@@ -105,15 +239,15 @@ The easiest way to checkout the source code for all the services is to run the p
 ```bash
 $ mkdir helidon-sock-shop
 $ cd helidon-sock-shop
-$ bash <(curl -s https://raw.githubusercontent.com/helidon-sock-shop/sockshop/master/update.sh)
+$ bash <(curl -s https://raw.githubusercontent.com/helidon-sock-shop/master/update.sh)
```

Comment:
Nope -- the script is within `sockshop` repo -- `helidon-sock-shop` is an org, not a repo, and can't have any files within it directly.